include/mbedtls/ssl.h

```diff
@@ -2817,9 +2817,9 @@ int mbedtls_ssl_set_hs_psk_opaque( mbedtls_ssl_context *ssl,
  *                 - \c mbedtls_ssl_context*: The SSL context to which
  *                                            the operation applies.
  *                 - \c const unsigned char*: The PSK identity
```

Comment:
In the case of the new client-side callback, this is actually the PSK identity _hint_, or am I misunderstanding something?

Reply:
I'm not sure, if I got your point. Which identity hint are you referring to? In my understanding, the client side callback receives the sever identity hint. For me the question is, how the client can configure the client identity.

Reply:
@peckto Yes that's right. My point is that now we're using a single callback which (a) for the client, receives a PSK identity hint, (b) for the server, receives a PSK identity.

Reply:
Thanks, now I see your point. You want to refer with `PSK identity hint` to what the client is sending and with `PSK identity` to what the server is sending. I will clarify this in the doc.

Reply:
I updated the doc accordingly